src/DotNet.Status.Web/Controllers/AzurePipelinesController.cs

```diff
@@ -145,7 +134,7 @@ private bool IsIgnoredStatus(Build build)
             }
         }
 
-        private async Task ProcessBuildNotificationsAsync(Build build)
+        private async Task ProcessBuildNotificationsAsync(IAzureDevOpsClient client, Build build)
```

Comment:
I'm not sure I love us passing the client around like this honestly. Seems incorrect

Reply:
The build doesn't know what org its in, and we can't create a client without knowing the org